app/controllers/api/v1/events_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Api::V1::EventsController < Api::V1::BaseController
   def show
-    event = Facebook::Event.new(params[:id])
-    event.load!
-    render json: event, status: 200
+    event = Facebook::Event.new(id: params[:id])
+    repository = Facebook::EventRepository.new
+    render json: repository.load!(event), status: 200
```

Comment:
And.. create a class method called `load` in `Facebook::EventRepository`, merging the line four into line five. It turns something like: `render json: Facebook::EventRepository.load(event)`. The status `200` is default.